GoogleForCreators/web-stories-wp
# Fixed error in video play/pause button after trimming

## Relevant Technical Choices

- Converted video node from function to state to make sure to re-run all effects when node updates

## User-facing changes

| Before | After |
|-|-|
| ![trimfix-before](https://user-images.githubusercontent.com/637548/208500583-f3c22ff8-5f12-4501-95c0-2fdd8e79b696.gif) | ![trim-fix-after](https://user-images.githubusercontent.com/637548/208500603-7ca5e8c9-4838-4e9a-ad96-e91088ff466e.gif) |

## Testing Instructions

This PR can be tested by following these steps:

1. Add a local video to canvas
2. Click play/pause and observe that play/pause button correctly change state when video state changes
3. Click trim
4. Exit trim
5. Click play/pause and observe that play/pause button correctly change state when video state changes

## Checklist




---

Fixes #11958
